PollBuddy-Server/backend/modules/utils.js

```diff
@@ -0,0 +1,35 @@
+const mongoConnection = require("../modules/mongoConnection.js");
+
+
+// Helper function for creating specified http response (https://pollbuddy.app/api/users)
+// error and data are optional
+function createResponse(isSuccess = true, error, data) {
+  const payload = {
+    result: isSuccess ? "success" : "failure"
+  };
+  if (error) {
+    payload.error = error;
+  }
+  if (data) {
+    payload.data = data;
+  }
+  return payload;
+}
+
+// Check if ID exists for a specific Collection
+// returns objectID if valid or null if nothing is found.
+async function validateID(collection, id) {
```

Comment:
As noted in the last meeting and in #424, I'll merge the best parts of the two validateID functions at merge time.